database.rules.json

```diff
@@ -2,10 +2,22 @@
   "rules": {
     ".read": false,
     ".write": false,
-    "messages": {
+    "games": {
       "123": {
-        ".read": true,
-        ".write": true
+        ".read": false,
+        ".write": false,
+        "chat": {
+          ".read": true,
+          ".write": true
+        },
+        "history": {
+          ".read": true,
+          ".write": true
+        },
+        "state": {
```

Comment:
What will this be used for?

Reply:
The live state of the current game, IE the current board position, whose turn it is, who's playing, who's spectating, etc